formats/JsonFormat.php

```diff
@@ -49,7 +49,7 @@ public function stringify(){
 			$entryTitle = $item->getTitle();
 			$entryUri = $item->getURI();
 			$entryTimestamp = $item->getTimestamp();
-			$entryContent = $this->sanitizeHtml($item->getContent());
```

Comment:
`? : '' ` will be cleaner

Reply:
What's going in here? Code review on yourself?

Reply:
Yeah, I made these changes locally but did not push them yet. And then I received a notification but did not have my computer with me.